pre_commit/clientlib/validate_base.py

```diff
@@ -3,12 +3,12 @@
 
 import argparse
 import os.path
-import pkg_resources
```

Comment:
heh, this was definitely my fault. at least i recognized that "someone" would make a dumb mistake like this!